js/ui/components/clubhouse.js

```diff
@@ -716,21 +723,48 @@ var ClubhouseOpenButton = new Lang.Class({
         this._highlightSoundItem = new SoundServer.SoundItem('clubhouse/entry/pulse');
         this._highlightLoopingSoundItem =
             new SoundServer.SoundItem('clubhouse/entry/pulse-loop');
+
+        this.connect('notify::visible',
+                     this._play_stop_highlighted_sound.bind(this));
+        this.connect('notify::highlighted',
+                     this._play_stop_highlighted_sound.bind(this));
     },
 
-    setHighlighted: function(highlighted) {
+    set highlighted(highlighted) {
```

Comment:
So you are converting the highlighted setter into a property. But should this be set at all if the button is not visible? A simpler approach would be to skip setting the highlighted if the button is hidden, which will also save playing an animation.

Reply:
I will opt for this. I was thinking that after completing the InitialContact puzzle the pulsating button would be shown unhighlighted but this is not the case.